packages/core/src/types.ts

```diff
@@ -2219,3 +2241,53 @@ export type PersistedMachineState<TState extends AnyState> = Pick<
     };
   };
 };
+
+export interface InspectedActorObject {
```

Comment:
this whole type is not used anywhere, can it be removed?

Reply:
Yes